src/main/java/org/terasology/launcher/util/DownloadUtils.java

```diff
@@ -188,6 +191,25 @@ public static URL createUrlJenkins(String jobName, int buildNumber, String subPa
         return new URL(urlBuilder.toString());
     }
 
+    public static boolean isJenkinsAvailable() {
+        logger.trace("Checking Jenkins availability...");
+        try {
+            HttpURLConnection conn = (HttpURLConnection) new URL(JENKINS_URL).openConnection();
+            try (AutoCloseable ac = conn::disconnect) {
+                conn.setConnectTimeout(JENKINS_TIMEOUT);
+                if (conn.getResponseCode() == HttpURLConnection.HTTP_OK) {
+                    logger.trace("Jenkins is available at {}", JENKINS_URL);
+                    return true;
+                } else {
+                    throw new ConnectException();
+                }
+            }
+        } catch (Exception e) {
```

Comment:
Please try to catch the most specific type of exception you can, e.g., `IOException` is better than just `Exception`. You could also use multiple `catch (…)` statements with the same `try`.

For instance, a `NullPointerException` would also be caught by this, although this is (hopefully) not intended here…